### PR TITLE
Don't use conda defaults channel

### PR DESCRIPTION
**Pull request by weiji14**
Prevent mixing conda-forge and defaults channel, should help reduce time to solve complex environment.

Really, just an excuse to test the `/condalock` command from #5.

### CODE REVIEW

File: conda-linux-64.lock [hunk 1]
```diff
@@ -1,2 +1,517 @@
...
+# pip fastjsonschema @ https://files.pythonhosted.org/packages/e4/be/cf1b876348070a23cb0c3ebfee7a452ad3a91b07b456dade3bd514656009/fastjsonschema-2.16.2-py3-none-any.whl#sha256=21f918e8d9a1a4ba9c22e09574ba72267a6762d47822db9add95f6454e51cc1c
+# pip py @ https://files.pythonhosted.org/packages/f6/f0/10642828a8dfb741e5f3fbaac830550a518a775c7fff6f04a007259b0548/py-1.11.0-py2.py3-none-any.whl#sha256=607c53218732647dff4acdfcd50cb62615cedf612e72d1724fb1a0cc6405b378
+# pip pywinpty @ https://files.pythonhosted.org/packages/22/77/59645ee028f410ef1d582fc2923d2cb61016ed38e6ae5022f405227ca2ad/pywinpty-2.0.9.tar.gz#sha256=01b6400dd79212f50a2f01af1c65b781290ff39610853db99bf03962eb9a615f
+# pip jupyter-desktop-server @ https://files.pythonhosted.org/packages/15/df/6f249cd3c3db684c8f9e67bc5470c4b593f882fbb03b1649f72ae7a26566/jupyter-desktop-server-0.1.3.tar.gz#sha256=5915a2dbbe250f59b3cc01859297b26b2e1836e03a5db4bfbfd438005649d7e4
+# pip jupyter-syncthing-proxy @ https://files.pythonhosted.org/packages/4d/53/fc5f8fc3fa21aa463ec8002b25187e9e565753b6b0caf8bb3b020022497e/jupyter_syncthing_proxy-1.0.3-py3-none-any.whl#sha256=f4550a7a4d22b97128743205a6ea001487f9f07c561befedaadc265670cc77f2
```
Comment (weiji14) [inline conda-linux-64.lock:517]: Ideally, we would use this `conda-linux-64.lock` to build the docker image for full reproducibility, but there are some PyPI-only packages right now that are not on conda-forge.

@yuvipanda, have you thought about creating conda-forge packages for https://github.com/yuvipanda/jupyter-desktop-server and https://github.com/yuvipanda/jupyter-syncthing-proxy?

File: environment.yml [hunk 2]
```diff
@@ -2,6 +2,7 @@ name: cryocloud
 
 channels:
   - conda-forge
+  - nodefaults
```
Comment (fperez) [inline environment.yml:5]: Nice! I didn't know about `nodefaults` :)

**Review by fperez — APPROVED** [review]

File: environment.yml (hunk 2 above)
Comment (weiji14) [reply environment.yml:5]: Ah yes, it's a little trick from https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html?highlight=nodefaults#creating-an-environment-file-manually :grin:

**Pull request merged**